jest.config.mjs

```diff
@@ -123,7 +123,7 @@ export default {
   // reporters: undefined,
 
   // Automatically reset mock state before every test
-  // resetMocks: false,
+  resetMocks: true,
```

Comment:
is there a specific reason to change the default?

Reply:
Yeah there was cross contamination between the mocks. I would have turned this on from the beginning if I knew it wasn't